### PR TITLE
logo fix

### PR DESCRIPTION
**Pull request by willsvankumc**
Testing evidence: 

image

### CODE REVIEW

**Comment by willsvankumc** [comment]
posting evidence regarding the apache folder removal shortly.

File: heron_wsgi/heron_srv.py [hunk 1]
```diff
@@ -770,21 +770,16 @@ def app_factory(global_config, **settings):
...
-        # In production use, static A/V media files would be
-        # served with apache, but for test purposes, we'll use
-        # paste DirectoryApp
-        # TODO: use paster
-        # app = prefix_router('/av/',
-        #                    fileapp.DirectoryApp(HeronAccessPartsApp.htdocs),
-        #                    )
+        app = prefix_router('/av/',
+                            fileapp.DirectoryApp(HeronAccessPartsApp.htdocs),
+                            )
```
Comment (dckc) [inline heron_wsgi/heron_srv.py]: I don't see any significant changes here. What am I missing?

Did you mean to commit these changes?

"Reviewing the actual diffs as you write the commit message is a good habit to establish. " -- UsingVersionControl

**Comment by willsvankumc** [comment]
@dckc Apologies for lack of context. 
These changes are to use paste to serve static image assets within the directory /av/ for the heron logo image. This entailed removing the comments for the paste functionality in heronserv.py and correcting the path in production.ini.example.

File: heron_wsgi/heron_srv.py (hunk 1 above)
Comment (dckc) [reply heron_wsgi/heron_srv.py]: oic... you un-commented some stuff.

**Comment by willsvankumc** [comment]
Heron-Admin appears to be functioning without the apache folder, and /templates/av/ is being served successfully with paste.

**Pull request merged**